### PR TITLE
Availability Support on Azure Firewall

### PR DESCRIPTION
**Pull request by tejasshah7**
## Description
Support for availability zones on azure firewall

## Checklist

- [Y] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [Y] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [Y] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [Y] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/Network/Network/ChangeLog.md [hunk 1]
```diff
@@ -21,6 +21,9 @@
 * Add DisableBgpRoutePropagation flag to Effective Route Table output
     - Updated cmdlet:
         - Get-AzEffectiveRouteTable
+* Updated below commands for feature: UseLocalAzureIpAddress flag on VpnConnection
+	- Updated New-AzVpnConnection: Added optional parameter -UseLocalAzureIpAddress to indicate that local azure ip address should be used as source address while initiating connection.
+	- Updated Set-AzVpnConnection: Added optional parameter -UseLocalAzureIpAddress to indicate that local azure ip address should be used as source address while initiating connection.
```
Comment (number213) [inline src/Network/Network/ChangeLog.md]: Please add a point about adding zone support to Azure Firewall

Comment (tejasshah7) [reply src/Network/Network/ChangeLog.md]: Added this.

**Comment by cormacpayne** [comment]
@tejasshah7 @number213 it looks like (almost) all of the Network tests are failing since the API version is different in the new SDK than the old one -- do we have updated recordings for version 20.0.0-preview of the SDK that we can pull into this branch?

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by number213** [comment]
@cormacpayne there is a PR to update SDK and test recordings
https://github.com/Azure/azure-powershell/pull/9250

**Comment by MikhailTryakhov** [comment]
@cormacpayne  can you please help to merge it?

**Pull request merged**